### PR TITLE
Rename Minetest to Luanti

### LINKED ISSUE #3164: Rename Minetest to Luanti in files
Rename Minetest to Luanti in the files.

### PR DESCRIPTION
**Pull request by Wuzzy2**
This PR changes the name "Minetest" to "Luanti" in all files. Fixes #3164.

It also changes the links from minetest.net to luanti.org domains (all tested).

Finally, the `game.conf` author is changed from "Minetest" to "Luanti". This is done in a separate commit because I'm not sure if doing so is safe. If this is a problem, just drop that commit.

### CODE REVIEW

**Comment by rollerozxa** [comment]
The author key is basically only used by ContentDB and doesn't really do anything without the release key right now. It's safe to change but will be overridden to "Minetest" when installing through the client as the MTG package on CDB is minetest/minetest_game.

File: game.conf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 title       = Minetest Game
-author      = Minetest
-description = A basic exploration, mining, crafting, and building, sandbox game with no NPCs, monsters, or animals. Minetest Game is usually used with mods added, and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
+author      = Luanti
```
Comment (sfan5) [inline game.conf:2]: yeah let's just remove it if it's useless anyway

File: mods/beds/README.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ See license.txt for license information.
 Authors of source code
 ----------------------
 Originally by BlockMen (MIT)
-Various Minetest developers and contributors (MIT)
+Various Luanti developers and contributors (MIT)
```
Comment (sfan5) [inline mods/beds/README.txt:8]: strictly speaking they were contributors to "Minetest" at the time
or more specifically "Minetest Game"?

**Comment by Wuzzy2** [comment]
Author removed from `game.conf`.

About the credits line: I thought the consensus was the rename applied retroactively, was it not?
Anyway, I want to hear other core dev opinions regarding the credits line, just to be sure.

**Comment by Wuzzy2** [comment]
Alright, I decided to replace the line in the credits to "Minetest Game developers".

**Pull request merged**